CMakeLists.txt

```diff
@@ -26,6 +35,14 @@ option(DF_USE_SSL "Provide support for SSL connections" ON)
 
 find_package(OpenSSL)
 
+if (SUPPORT_ASAN)
+  # set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -fsanitize=address")
```

Comment:
why it's commented out under macro?

Reply:
because it does not work yet. I will enable it in the subsequent PRs.